ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -115,12 +120,24 @@ tests:
   skip_if_only_changed: ^(?:docs|\.github)/|\.md$|^(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
   steps:
     cluster_profile: aws
-    env:
-      OO_CHANNEL: preview
-      OO_INSTALL_NAMESPACE: openshift-windows-machine-config-operator
-      OO_PACKAGE: windows-machine-config-operator
-      OO_TARGET_NAMESPACES: '!install'
     test:
+    - as: install
+      cli: latest
+      commands: |
+        oc create namespace openshift-windows-machine-config-operator
+        oc label --overwrite=true ns openshift-windows-machine-config-operator openshift.io/cluster-monitoring=true \
+        security.openshift.io/scc.podSecurityLabelSync=true \
+        pod-security.kubernetes.io/enforce=privileged
+        operator-sdk run bundle --timeout=10m --security-context-config restricted -n openshift-windows-machine-config-operator "$OO_BUNDLE"
```

Comment:
- PS: I don't understand the doc as I can't figure out where `$OO_BUNDLE_INIT` is defined in the [example](https://docs.ci.openshift.org/docs/how-tos/testing-operator-sdk-operators/#operator-upgrade-testing).
- It is not clear to me how this command results in the previous operator version to be installed

Reply:
The upgrade testing isn't included in this PR, but I can explain what needs to be done according to the upgrade example.
`$OO_BUNDLE_INIT` is defined by
```
      dependencies:
      - env: OO_BUNDLE_INIT
        name: my-bundle-init
```

This references the base image:
```
base_images:
  my-bundle-init:
    name: "my-bundle"
    namespace: my-namespace
    tag: init
```

This isn't super clear by the example but what can be done is something like:
```
base_images:
  wmco-bundle-init:
    name: "4.13"
    namespace: ocp
    tag: wmco-bundle
```

`base_images` can reference any image that has been promoted. We can add the `wmco-bundle` image to the `promotion` block. This ends up creating a post-submit job after each successful PR. So, in a 4.14 job we can use the above block to make use of the last merged 4.13 bundle.

Reply:
I suggest documenting this in the commit.

Reply:
aded some info in the second commit's message